src/components/docs/Navigation.js

```diff
@@ -59,7 +59,7 @@ const Label = styled(props => <span {...props} />)`
   border: 1px solid;
   border-radius: 2px;
   text-transform: uppercase;
-  font-size: 0.666666667rem;
+  font-size: 0.95rem;
```

Comment:
this could probably be `0.75rem` and look better

current: 
<img width="195" alt="Screen Shot 2019-08-02 at 5 28 41 PM" src="https://user-images.githubusercontent.com/4924860/62399790-06a28680-b54b-11e9-804d-14e88422254b.png">


with change:
<img width="196" alt="Screen Shot 2019-08-02 at 5 28 46 PM" src="https://user-images.githubusercontent.com/4924860/62399786-04402c80-b54b-11e9-9de3-743252795b15.png">